### PR TITLE
doc：update install-dependencies.md

### PR DESCRIPTION
**Pull request by htwdjqr**
### What this PR does / why we need it:
update install-dependencies.md

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
Do we need to update the English version?

File: doc/zh-cn/install-dependencies.md [hunk 1]
```diff
@@ -40,15 +40,21 @@ CentOS 7
 wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 sudo rpm -ivh epel-release-latest-7.noarch.rpm
 
+# 安装 etcd
```
Comment (juzhiyuan) [inline doc/zh-cn/install-dependencies.md:43]: Please update this doc in English, too.

Comment (htwdjqr) [reply doc/zh-cn/install-dependencies.md:43]: It is copied from the English document,no need to update the English version.

**Review by juzhiyuan — COMMENTED** [review]
LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Comment by membphis** [comment]
@htwdjqr many thx for your contribution ^_^

**Pull request merged**